### ISSUE TITLE
Introduction of "namespace types" in @types/node@10.* breaks compatibility with Node.js LTS version 8.11.*

### ISSUE THREAD

**Issue by DavidAnson** [DefinitelyTyped/DefinitelyTyped#25861]
The problematic commit seems to be https://github.com/DefinitelyTyped/DefinitelyTyped/commit/bffb03282272b37ccb12429026f4504a39a3cd83 by @rbuckton 

```
C:\T\oops>node --version
v8.11.1

C:\T\oops>npm i typescript @types/node

+ typescript@2.8.3
+ @types/node@10.1.1

C:\T\oops>type empty.ts

C:\T\oops>node_modules\.bin\tsc empty.ts
node_modules/@types/node/index.d.ts(6207,55): error TS2304: Cannot find name 'Map'.
node_modules/@types/node/index.d.ts(6214,55): error TS2304: Cannot find name 'Set'.
node_modules/@types/node/index.d.ts(6218,64): error TS2304: Cannot find name 'Symbol'.
node_modules/@types/node/index.d.ts(6224,59): error TS2304: Cannot find name 'WeakMap'.
node_modules/@types/node/index.d.ts(6225,59): error TS2304: Cannot find name 'WeakSet'.

C:\T\oops>npm i @types/node@9.6.15

+ @types/node@9.6.15

C:\T\oops>node_modules\.bin\tsc empty.ts

C:\T\oops>
```

**Comment by DavidAnson** [comment]
If this is deliberate because it's expected the major version of this package will match the version of Node, I understand. But it may be difficult to enforce on larger projects where contributors have both 8.* and 10.* installed.

**Comment by Flarna** [comment]
You use @types/node in version 10.1.1 which matches to NodeJS >=10.1 but it seems you want 8.x. Therefore I think you have to adapt your package.json to select the correct version.
To fix above missing symbols add/use `es2015` as lib in tsconfig.json. I guess you have `es5` now where above types  are not yet included.

**Comment by rbuckton** [comment]
This should be addressed by #25882.
